iceoryx_hoofs/test/moduletests/test_posix_signal_handler.cpp

```diff
@@ -66,7 +66,7 @@ class SignalHandler_test : public Test
 
     void registerSignal(const int signal, void (*const callback)(int))
     {
-        struct sigaction action;
+        struct sigaction action = {};
```

Comment:
Please initialize it like `struct sigaction action{};`

Reply:
With that, clang-format will do it that way:
```cpp
        struct sigaction action
        {
        };
```

The other proposal to remove the `struct` keyword is not applicable everywhere.
Using ` = {};` might look a bit redundant but we avoid using `// clang-format`. I would prefer to use it that way consistently.